app/metrics.yaml

```diff
@@ -123,6 +123,19 @@ events:
     notification_emails:
     - fenix-core@mozilla.com
     expires: "2020-03-01"
+  total_uri_count:
```

Comment:
I wanted to put this under the metrics category, but this bug got in the way:
https://github.com/mozilla-mobile/android-components/issues/2793